CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## Unreleased
 
```

Comment:
All looks good! Just make sure to add an entry to CHANGELOG. This is a minor change and we need to keep a track of our changes.